### PR TITLE
initial commit - add multiple correct answer functionality

### PR DESCRIPTION
**Pull request by npreiber**
add a list of possible correct answers

### CODE REVIEW

File: plugins/input-exercise-renderer/src/index.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const inputExerciseRendererPlugin: RendererPlugin<
 }
 
 export interface InputExercisePluginState {
-  correctValue: string
+  correctAnswers: CorrectAnswer[]
```
Comment (inyono) [inline plugins/input-exercise-renderer/src/index.ts]: Does this need to be an object?

File: plugins/input-exercise/src/editor.tsx [hunk 2]
```diff
@@ -48,14 +49,41 @@ export class InputExerciseEditor extends React.Component<
...
-    const { onChange } = this.props
-    onChange({
-      correctValue: value
-    })
-  }
+
+	addCorrectAnswer = () => {
+		const { onChange, state } = this.props
+		const { correctAnswers } = state
+		console.log(correctAnswers)
```
Comment (inyono) [inline plugins/input-exercise/src/editor.tsx]: remnant `console.log`

File: plugins/input-exercise/src/editor.tsx [hunk 3]
```diff
@@ -95,32 +123,53 @@ export class InputExerciseEditor extends React.Component<
...
+							)
+						})}
+           	<button onClick={this.addCorrectAnswer}>
+           		Richtige Antwort hinzufügen
+           	</button>
+          </React.Fragment>
+				)}
         {readOnly ? (
-          <InputExerciseRenderer {...this.props} />
+				null
```
Comment (inyono) [inline plugins/input-exercise/src/editor.tsx]: you can combine that with the previous `if`

File: plugins/input-exercise-renderer/src/index.ts (hunk 1 above)
Comment (npreiber) [reply plugins/input-exercise-renderer/src/index.ts]: no this does not need to be an object.  I made it an object in anticipation of a need to allow  feedback for various accepted answers. If we revert to string there will have to be a number of alterations throughout plugin

Comment (inyono) [reply plugins/input-exercise-renderer/src/index.ts]: Fair enough 👍

File: plugins/input-exercise/src/editor.tsx (hunk 2 above)
Comment (npreiber) [reply plugins/input-exercise/src/editor.tsx]: This change made in recent commit.

File: plugins/input-exercise/src/editor.tsx (hunk 3 above)
Comment (npreiber) [reply plugins/input-exercise/src/editor.tsx]: This change was made in the recent commit.

**Pull request merged**